frame/composable-support/src/validation.rs

```diff
@@ -144,43 +233,62 @@ mod test {
 		b: u32,
 	}
 
-	impl Validate<ValidARange> for X {
-		fn validate(self) -> Result<X, &'static str> {
-			if self.a > 10 {
+	impl Validate<X, ValidARange> for ValidARange {
+		fn validate(extrinsic_input: X) -> Result<X, &'static str> {
```

Comment:
BTW it's an abstraction to validate a structure, not specifically an extrinsic input, since it's fully generic, the name is meaningless isn't it :P?

Reply:
That's a good point. I've cleaned up the name here 120020d33c59a904453d831923a20b7c1064e5c3